good-enough-practices-for-scientific-computing.tex

```diff
@@ -920,13 +940,7 @@ \subsection*{Version Control Systems}
 projects and for users in academia and nonprofits.
 \withurl{GitLab}{https://about.gitlab.com} is a well-regarded
 alternative that some prefer, because the GitLab platform itself is
-free and open source.
-
-For those who find Git's command-line syntax inconsistent
-and confusing, \withurl{Mercurial}{https://www.mercurial-scm.org/} is
-a good choice; \withurl{Bitbucket}{https://bitbucket.org/} provides
-free hosting for both Git and Mercurial repositories, but does not
-have nearly as many scientific users.
+free and open source. 
```

Comment:
I'm OK with removing mention of Mercurial, but think it's unfair to remove mention of BitBucket.